kubedog.go

```diff
@@ -50,6 +50,7 @@ func (kdt *Test) Run() {
 	kdt.scenarioContext.Step(`^a Kubernetes cluster$`, kdt.KubeContext.AKubernetesCluster)
 	kdt.scenarioContext.Step(`^(?:I )?(create|submit|delete) (?:the )?resource ([^"]*)$`, kdt.KubeContext.ResourceOperation)
 	kdt.scenarioContext.Step(`^(?:I )?(create|submit|delete) (?:the )?resources in ([^"]*)$`, kdt.KubeContext.MultiResourceOperation)
+	kdt.scenarioContext.Step(`^(?:I )?(create|submit|delete) (?:the )?resources in ([^"]*) in (?:the )?([^"]*) namespace`, kdt.KubeContext.MultiResourceOperationInNamespace)
```

Comment:
Would appreciate some attention here. Not sure if step syntax makes sense here.

Reply:
I think you need the `$` at the end, not sure tho

Reply:
I only see that for `([^"]*)`. Since it ends with "namespace", I think it might be okay without `$`.

Reply:
Tested out locally and seems Gherkin was getting mapped to correct step/func.